File: Python/person_analysis.py
class person_analysis:
    def __init__(self, name, age, height, weight):
        self.name = name
        self.age = age
        self.height = height
        self.weight = weight

    def calculate_bmi(self):
        return self.weight / ((self.height / 100) ** 2)

    def get_health_status(self):
        bmi = self.calculate_bmi()
        if bmi < 18.5:
            return "Underweight"
        elif 18.5 <= bmi < 24.9:
            return "Normal weight"
        elif 25 <= bmi < 29.9:
            return "Overweight"
        else:
            return "Obesity"

    def __str__(self):
        return (f"Name: {self.name}, Age: {self.age}, Height: {self.height}, Weight: {self.weight}, "
                f"BMI: {self.calculate_bmi()}, Health Status: {self.get_health_status()}")

File: PHP/person_analysis.php
<?php
class person_analysis {
    private $name;
    private $age;
    private $height;
    private $weight;
    public function __construct($name, $age, $height, $weight) {
        $this->name = $name;
        $this->age = $age;
        $this->height = $height;
        $this->weight = $weight;
    }
    public function calculateBMI() {
        return $this->weight / (($this->height / 100) ** 2);
    }
    public function getHealthStatus() {
        $bmi = $this->calculateBMI();
        if ($bmi < 18.5) {
            return "Underweight";
        } elseif ($bmi >= 18.5 && $bmi < 24.9) {
            return "Normal weight";
        } elseif ($bmi >= 25 && $bmi < 29.9) {
            return "Overweight";
        } else {
            return "Obesity";
        }
    }
    public function __toString() {
        return "Name: $this->name, Age: $this->age, Height: $this->height, Weight: $this->weight, BMI: " . $this->calculateBMI() . ", Health Status: " . $this->getHealthStatus();
    }
}
?>
